backend/hyperglosae/src/views/lib/links.js

```diff
@@ -1,36 +1,55 @@
 exports.parseReference = parseReference = (reference) => {
-  let [id, fragment] = reference.split('#');
-  return {id, fragment};
-}
+  let [id, fragment] = reference.split("#");
+  return { id, fragment };
+};
```

Comment:
These lines are not related to your contribution please don't include them in your commits.